vsys-mainnet.conf

```diff
@@ -69,6 +69,15 @@ vsys {
 
     # Hash of API key string
     api-key-hash = "your own api key's hash"
+
+    # Enable/disable custom http api routes
+    custom-api {
+      # Routes with "/transactions"
+      transactions {
+        address-transaction-count = off
+        address-transaction-list = on
+      }
+    }
```

Comment:
let's make the finial decision, we may delete the custom setting in conf, land make this as hidden setting for general user.